tests/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilderTest.php

```diff
@@ -300,19 +300,22 @@ public function testSetWhereWithSameName()
             $this->doctrineListBuilder->where($fieldDescriptors[$key], $value);
         }
 
-        $this->assertCount(2, PHPUnit_Framework_Assert::readAttribute($this->doctrineListBuilder, 'whereValues'));
-        $whereValues = PHPUnit_Framework_Assert::readAttribute($this->doctrineListBuilder, 'whereValues');
-        $this->assertEquals(3, $whereValues['title_id']);
-        $this->assertEquals(1, $whereValues['desc_id']);
-
-        $this->assertCount(2, PHPUnit_Framework_Assert::readAttribute($this->doctrineListBuilder, 'whereFields'));
-        $whereFields = PHPUnit_Framework_Assert::readAttribute($this->doctrineListBuilder, 'whereFields');
-        $this->assertEquals($fieldDescriptors['title_id'], $whereFields['title_id']);
-        $this->assertEquals($fieldDescriptors['desc_id'], $whereFields['desc_id']);
-
+        $this->assertExpressions();
         $this->doctrineListBuilder->execute();
     }
 
+    public function assertExpressions()
```

Comment:
I would not extract this assertions into an own method, because both methods in these tests don't do the same. Currently you have removed the check for the comparator in this method, if you introduce it again both tests have different results.